CHANGELOG.rst

```diff
@@ -146,7 +146,8 @@ time or change existing behaviors in order to make them less surprising/more use
   Thanks to `@anntzer`_ for the PR.
 
 
-*
+* Better message in case of not using parametrized variable (see `#1539`_).
```

Comment:
I think the message looks good, I just think it should go under the `Changes` section instead of `New Features`.